x/superfluid/keeper/stake.go

```diff
@@ -503,17 +504,31 @@ func (k Keeper) forceUndelegateAndBurnOsmoTokens(ctx sdk.Context,
 	if err != nil {
 		return err
 	}
-	// TODO: Better understand and decide between ValidateUnbondAmount and SharesFromTokens
-	// briefly looked into it, did not understand what's correct.
-	// TODO: ensure that intermediate account has at least osmoAmount staked.
```

Comment:
Were these TODOs removed intentionally?

Reply:
yes, I've erased this since we have a safety bound of ensuring intermediate account has at least the osmo Amount staked (from the delegation check + and it would error anyways on the `InstantUndelegate` function as well)